libraries/AP_BattMonitor/LogStructure.h

```diff
@@ -19,6 +19,7 @@
 // @Field: Res: estimated battery resistance
 // @Field: RemPct: remaining percentage
 // @Field: H: health
+// @Field: SH: state of health percentage
```

Comment:
note the invalid value, currently 255, possibly 0 (https://github.com/ArduPilot/ardupilot/pull/25926/files#r1445152646). 
```suggestion
// @Field: SH: state of health percentage, 255 if unknown
```

Reply:
I've added, "0 if unknown".  txs again